tests/UI/pages/BO/orders/view/viewOrderPage.ts

```diff
@@ -0,0 +1,10 @@
+import ViewOrderBasePage from '@pages/BO/orders/view/viewOrderBasePage';
+
+/**
+ * @class
+ * @extends ViewOrderBasePage
+ */
+class ViewOrderPage extends ViewOrderBasePage {
```

Comment:
Maybe something to improve later : 
```js
// viewOrderBasePage.ts
const viewOrderBasePage = new ViewOrderBasePage();
export {viewOrderBasePage, ViewOrderBasePage};
```

1. Import `ViewOrderBasePage` in pages to extend it
2. Import `viewOrderBasePage` in test to use it

So you can delete this file

Reply:
Fixed